model/metrics/system-metrics.yaml

```diff
@@ -490,3 +490,13 @@ groups:
     brief: "Total number of processes created over uptime of the host"
     instrument: counter
     unit: "{process}"
+
+  # system.linux.* metrics
+  - id: metric.system.linux.memory.available
+    type: metric
+    metric_name: system.linux.memory.available
+    brief: "An estimate of how much memory is available for starting new applications, without swapping"
+    note: |
+      Available since Linux 3.14: see `MemAvailable` in [here](https://man7.org/linux/man-pages/man5/proc.5.html).
```

Comment:
I see at https://github.com/open-telemetry/semantic-conventions/issues/257#issuecomment-1678926537 the background dicsussions on why to make this Linux specific.

However:
1) Do we have any plan/suggestion of what would be the equivalent for other operating systems? What we should suggest to our users to use instead of this?
2) If we finally keep the linux specific one we need to clearly state here in the [note the purpose of this metric and why it exists solely for Linux](https://github.com/open-telemetry/semantic-conventions/issues/257#issuecomment-1678926537).

Reply:
1. Just `state=free` value from https://github.com/open-telemetry/semantic-conventions/blob/47d7a1ed8080117b30020c949dddc4cff118298d/docs/system/system-metrics.md#metric-systemmemoryusage, nothing fancy
2. Ah, indeed, I did not do that. Can I just link https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19149#issuecomment-1467634451 in the note? Or do I need to paste/rephrase it? (I guess the latter is better)

Reply:
I would mention the `state=free` alternative in the note along with a summary from https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19149#issuecomment-1467634451. If we want to additionally add the link from the issue it would be fine but I think that the note should be enough on it's own (with or without the link).

Reply:
Got it. I updated the note, please take a look.